MarsAdvancedTaskPart1NUnitAutomationV3/Steps/AboutMeSteps.cs

```diff
@@ -0,0 +1,84 @@
+﻿using MarsAdvancedTaskPart1NUnitAutomation.Pages.ProfileOverview.ProfileAboutMeComponent;
+using MarsAdvancedTaskPart1NUnitAutomation.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsAdvancedTaskPart1NUnitAutomation.Steps
+{
+
+    public class AboutMeSteps : CommonDriver
+    {
+        string actualMessage = string.Empty;
+        string expectedMessage = string.Empty;
+
+        AboutMeEditComponent aboutMeEditComponent = new AboutMeEditComponent();
```

Comment:
can you please use constructor, please fix this in all other classes

Reply:
I will do it.